src/sentry/api/endpoints/project_app_store_connect_credentials.py

```diff
@@ -238,6 +219,7 @@ def post(self, request, project, credentials_id):
             credentials.update(new_credentials)
 
             credentials["encrypted"] = encrypt.encrypt_object(secrets, key)
+            credentials["refreshDate"] = datetime.utcnow()
             credentials["id"] = uuid4().hex
```

Comment:
Not a good idea to update the "refreshDate" here. A user might change the application id or ,  the appConnect info... all of these are not going to touch the itunes session, which is still going to expire after two weeks since it was obtained. 

A safer way to do it is to put the refreshDate into the sessionContext (at the end of 2fa) and then here if you can find a refreshDate it means that you just did a 2fa and you can refresh the date, if you don't find a "refreshDate" in the sessionContext that you shouldn't update it.